components/playground.tsx

```diff
@@ -336,48 +547,58 @@ export default function Playground({
 							</FormItem>
 						)}
 					/>
-					{userId && guildMember !== undefined ? (
-						<FormField
-							control={form.control}
-							name="sendToDb"
-							render={({ field }) => (
-								<FormItem>
-									<FormLabel>Register this passport</FormLabel>
-									<FormDescription>
-										{registerCheckboxDisabled
-											? "Sorry, we're at capacity for this upcoming passport ceremony :("
-											: "Sign up for the passport ceremony this Friday at 10pm at Hack Night. By checking this box you pinky promise you will ACTUALLY be there this Friday. If you can't make it please DM Matthew so we don't prep for you!"}
-									</FormDescription>
-									<FormControl>
-										<Checkbox
-											className="h-6 w-6"
-											disabled={registerCheckboxDisabled}
-											onCheckedChange={(e) => {
-												// it hasn't updated the value at this point yet, so it's the opposite actually
-												if (field.value === true) {
-													setGenerationSteps(GENERATION_STEPS.base);
-												} else {
-													setGenerationSteps(GENERATION_STEPS.register);
-												}
-												return field.onChange(e);
-											}}
-											checked={field.value}
-										/>
-									</FormControl>
-									<FormMessage />
-								</FormItem>
-							)}
-						/>
-					) : null}
-					<Button className="amberButton" type="submit" disabled={isLoading}>
-						{isLoading ? "Generating..." : "Generate"}
-					</Button>
+					{form.getValues("sendToDb") == "true" ? (
+						<Dialog>
+							<DialogTrigger
+								className="rounded-[0.25rem] border-2 border-slate-800 bg-amber-500 px-4 py-2 text-sm font-bold text-slate-800 shadow-[4px_4px_0_0_#0f172a] transition-colors hover:bg-amber-500/90 focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-white focus-visible:ring-offset-0"
+								disabled={isLoading}
+							>
+								{isLoading ? "Generating..." : "Generate"}
+							</DialogTrigger>
+							<DialogContent className="w-11/12 sm:w-auto overflow-y-auto">
+								<DialogHeader>
+									<DialogTitle className="text-2xl">
+										Passport Ceremony Registration
+									</DialogTitle>
+								</DialogHeader>
+								<DialogDescription className="flex flex-col gap-2 leading-relaxed text-base">
```

Comment:
<img width="594" alt="Screenshot 2024-09-24 at 9 06 32 PM" src="https://github.com/user-attachments/assets/1b924080-97ce-467f-bf26-7ce8fa9a72b9">

Idk I still feel like this needs less of a gap between the heading and description, like one or two lines less

Reply:
Shortened a bit more in [4eed739](https://github.com/purduehackers/passport-issuing-office/pull/52/commits/4eed7391fb891304bafd230079d42a0cb532328c), see if that's better

Reply:
<img width="641" alt="Screenshot 2024-09-24 at 9 57 35 PM" src="https://github.com/user-attachments/assets/c3c27213-3618-4d0b-9d22-d6879e039714">

Looks great now